x/auth/client/txbuilder/txbuilder_test.go

```diff
@@ -55,7 +55,7 @@ func TestTxBuilderBuild(t *testing.T) {
 				Sequence:      1,
 				Memo:          "hello",
 				Msgs:          defaultMsg,
-				Fee:           auth.NewStdFee(100, sdk.NewCoin("steak", sdk.NewInt(1))),
+				Fee:           auth.NewStdFee(100, sdk.NewCoin("stake", sdk.NewInt(1))),
```

Comment:
Should we update this too?

Reply:
That shouldn't matter too much. One can put anything instead of stake

Reply:
Just for consistency I suppose. Every other test was updated.

Reply:
Done